README.md

```diff
@@ -49,9 +52,9 @@ You can chose between using a [programming language template](https://github.com
 
 **Templates**
 
-Command: `faas-cli new FUNCTION_NAME --lang python/node/ruby/Dockerfile/etc`
+Command: `faas-cli new FUNCTION_NAME --lang python/node/go//ruby/Dockerfile/etc`
```

Comment:
Are there supposed to be two `/` characters after `go`?

Reply:
I think you're looking at an old revision there @ericstoekl ?